sqlmesh/core/context.py

```diff
@@ -621,7 +642,15 @@ def render(
 
         model = self.get_model(model_or_snapshot, raise_if_missing=True)
 
-        expand = self.dag.upstream(model.name) if expand is True else expand or []
+        if expand and not isinstance(expand, bool):
+            expand = {
+                normalize_model_name(
+                    x, default_catalog=self.default_catalog, dialect=self.config.dialect
```

Comment:
Shouldn't this be a model dialect since it refers to a model name?

Reply:
if expand is passed in by a sure, it should be the config, but if it comes from the dag, then it should already be normalized to the specifications of the model, so you shouldn't renormalize it with the config dialect.

Reply:
Since this is a context method then it should be the config dialect